### PR TITLE
Client install instructions and code toggle improvements

### LINKED ISSUE #6698: Update Dev Guide code samples to be more vertically compact
Got some feedback that the code in Developer Guide pages like [Insert Data](http://cockroachlabs.com/docs/v20.1/insert-data.html) should be visible directly below the tabs.

In other words: put the code samples closer to the heading buckets so that developers can see the code on the same page.

### LINKED ISSUE #6703: Create task-oriented driver/ORM install instructions
@jseldess brought this up in a separate PR:
https://github.com/cockroachdb/docs/pull/6691#pullrequestreview-365287985

We currently have no instructions for specific drivers/ORMs outside of their Hello World tutorials. We should look into creating task-oriented driver/ORM install instructions.

### PR DESCRIPTION
**Pull request by jseldess**
This PR refactors to "supported drivers and orms" page to provide installation instructions and related links for each tool. It also improves the UX of code toggles throughout the guide. 

Fixes #6698.
Fixes #6703.

### CODE REVIEW

File: v19.2/developer-guide-overview.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ For instructions showing how to do a specific task, see the links below.
 
 ## Common Tasks
 
+- [Install a Posgres Client](install-client-drivers.html)
```
Comment (rmloveland) [inline v19.2/developer-guide-overview.md]: typo: "postgres client"

**Comment by awoods187** [comment]
This looks great! Two small minor nits
Feel free to ignore this but I think the language boxes should be the same size:
image
Also, the text in the boxes gets smaller when you make the window smaller but the text on the page doesn't change:
image

**Comment by awoods187** [comment]
One other thought, should we say driver after things like `pg`?

**Comment by awoods187** [comment]
Also, the following items should not be beta:
Go pq driver
GORM
TypeORM

**Pull request merged**